src/Kriti/Eval.hs

```diff
@@ -22,13 +23,17 @@ data EvalError
   = InvalidPath B.ByteString Span (V.Vector Accessor)
   | TypeError B.ByteString Span T.Text
   | RangeError B.ByteString Span
+  | FuncNotFound B.ByteString Span T.Text
```

Comment:
I think we are going to want to lookup the function(s) in the parsing step

Reply:
This would be cumbersome but would be really cool to have this. I will add a note in the PR description. Marking this as resolved as now.